### PR TITLE
Add profile zipkin, consul

### PR DESCRIPTION
**Pull request by Warchant**
Needed to register didresolver in consul (needed by api service)

### CODE REVIEW

File: src/main/resources/bootstrap.yml [hunk 1]
```diff
@@ -23,3 +27,20 @@ spring:
     enabled: true
     sampler:
       probability: 1.0
+---
+spring:
+  profiles: with_consul
+  cloud:
+    consul:
```
Comment (tdorzhi) [inline src/main/resources/bootstrap.yml]: we should disable spring.cloud.consul.config.enabled=false

Comment (Warchant) [reply src/main/resources/bootstrap.yml]: config is not in gradle dependencies

Comment (Warchant) [reply src/main/resources/bootstrap.yml]: but just to be sure I will add this

**Comment by Warchant** [comment]
Closing for now.

**Pull request closed without merging**